src/cli/cli.ts

```diff
@@ -136,6 +137,22 @@ program
       });
     });
 
+if (process.env.PWTRACE) {
```

Comment:
This is our way of marking this as experimental?

Reply:
Yep.